<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLPreferenceDialog</class>
 <widget class="QDialog" name="SLPreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="cameraLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera:</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Projector:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="patternModeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="patternModeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="projectorComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="triggerModeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Mode:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="triggerModeLayout">
    <item>
     <widget class="QRadioButton" name="triggerHardwareRadioButton">
      <property name="text">
       <string>Hardware</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="triggerSoftwareRadioButton">
      <property name="text">
       <string>Software</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="writeToDiskLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Write to disk:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="shutterLayout">
    <item>
     <widget class="QLabel" name="shutterLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Shutter:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="shutterDoubleSpinBox">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="delayMsLabel_2">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="triggerLayout">
    <item>
     <layout class="QHBoxLayout" name="shiftLayout">
      <item>
       <widget class="QLabel" name="shiftLabel">
        <property name="text">
         <string>Shift:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shiftSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="delayLayout">
      <item>
       <widget class="QLabel" name="delayLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Delay:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delaySpinBox">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="delayMsLabel">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="writeToDiskLayout">
    <item>
     <widget class="QCheckBox" name="pointCloudsCheckBox">
      <property name="text">
       <string>Point Clouds</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="trackingCheckBox">
      <property name="text">
       <string>Tracking Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="diamondPatternLayout">
    <item>
     <widget class="QCheckBox" name="diamondPatternCheckBox">
      <property name="text">
       <string>Diamond Pixel Pattern</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="aquisitionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Aquisition:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="aquisitionLayout">
    <item>
     <widget class="QRadioButton" name="aquisitioncontinuousRadioButton">
      <property name="text">
       <string>Continuous</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="aquisitionSingleRadioButton">
      <property name="text">
       <string>Single Point Cloud</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="patternModeLayout">
    <item>
     <widget class="QCheckBox" name="patternHorizontalCheckBox">
      <property name="text">
       <string>Horizontal</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="patternVerticalCheckBox">
      <property name="text">
       <string>Vertical</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>cameraLabel</zorder>
  <zorder>projectorLabel</zorder>
  <zorder>cameraComboBox</zorder>
  <zorder>patternModeLabel</zorder>
  <zorder>patternModeComboBox</zorder>
  <zorder>projectorComboBox</zorder>
  <zorder>triggerModeLabel</zorder>
  <zorder>writeToDiskLabel</zorder>
  <zorder>aquisitionLabel</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SLPreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SLPreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
